### PR TITLE
feat(media-timeout): add more metrics

### PR DESCRIPTION
**Pull request by torgeadelin**
# COMPLETES #< INSERT LINK TO ISSUE >
Helps: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-435515
## This pull request addresses
-add more behavioural metrics

## by making the following changes

- add new behavioural metric, improve media failed metric

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

-unit, manual tests

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/plugin-meetings/src/meeting/index.ts [hunk 1]
```diff
@@ -5728,6 +5728,26 @@ export default class Meeting extends StatelessWebexPlugin {
...
+          correlation_id: this.correlationId,
+          locus_id: this.locusUrl.split('/').pop(),
+          reason: error.message,
+          stack: error.stack,
+          code: error.code,
+          turnDiscoverySkippedReason,
+          turnServerUsed,
+          isMultistream: this.isMultistream,
+          signalingState:
+            this.mediaProperties.webrtcMediaConnection?.multistreamConnection?.pc?.pc
```
Comment (marcin-bazyl) [inline packages/@webex/plugin-meetings/src/meeting/index.ts:5741]: what about non-multistream case? we could also send the same things for transcoded meetings

File: packages/@webex/plugin-meetings/src/meetings/util.ts [hunk 2]
```diff
@@ -49,6 +51,16 @@ MeetingsUtil.handleRoapMercury = (envelope, meetingCollection) => {
     if (meeting) {
       const {seq, messageType, tieBreaker, errorType, errorCause} = data.message;
 
+      Metrics.sendBehavioralMetric(BEHAVIORAL_METRICS.ROAP_MERCURY_EVENT_RECEIVED, {
+        correlation_id: data.correlationId,
+        seq,
+        message_type: messageType,
+        sdp: data.message.sdps?.length > 0 ? data.message.sdps[0] : undefined,
```
Comment (marcin-bazyl) [inline packages/@webex/plugin-meetings/src/meetings/util.ts:58]: SDP can be quite big, do we really need it and is it fine to be sending such long strings with metrics?

File: packages/@webex/plugin-meetings/src/meetings/util.ts [hunk 3]
```diff
@@ -49,6 +51,16 @@ MeetingsUtil.handleRoapMercury = (envelope, meetingCollection) => {
...
+        tie_breaker: tieBreaker,
```
Comment (marcin-bazyl) [inline packages/@webex/plugin-meetings/src/meetings/util.ts:59]: tie breaker for sure won't be needed for anything in metrics

Comment (torgeadelin) [reply packages/@webex/plugin-meetings/src/meetings/util.ts:59]: removed

File: packages/@webex/plugin-meetings/src/meetings/util.ts (hunk 2 above)
Comment (torgeadelin) [reply packages/@webex/plugin-meetings/src/meetings/util.ts:58]: yeah, i noticed while manual testing, removed it

File: packages/@webex/plugin-meetings/src/meeting/index.ts (hunk 1 above)
Comment (torgeadelin) [reply packages/@webex/plugin-meetings/src/meeting/index.ts:5741]: where can i find those?

**Review by marcin-bazyl — APPROVED** [review]

**Pull request merged**